hieradata/dev/wso2/wso2is/pattern-1/default.yaml

```diff
@@ -58,6 +57,8 @@ wso2::template_list:
 #  - repository/conf/security/cipher-tool.properties
 #  - repository/conf/security/cipher-text.properties
 #  - repository/conf/security/secret-conf.properties
+
+#wso2::file_list_copy_without_refresh:
```

Comment:
I think it would be better to rename this variable to something similar to "service_refresh_file_list" and explain its behaviour in a comment.